templates/docs/examples/templates/_card-fixed-width.html

```diff
@@ -0,0 +1,33 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Card / Default{% endblock %}
+
+{% block standalone_css %}patterns_card{% endblock %}
+
+{% block test_content %}
+<!-- <div class="p-card">
```

Comment:
Remove stuff that is not needed.

Reply:
removed

Reply:
@lyubomir-popov Doesn't seem removed

Reply:
oh sorry wrong file, this is actually not used, removed the entire file